client/src/components/Tickets/TicketModal.js

```diff
@@ -170,6 +191,9 @@ const TicketHistoryCard = ({
       </CardHeader>
       <Collapse isOpen={isOpen}>
         <CardBody>{content}</CardBody>
+        <CardBody>
+          <Attachments attachments={attachmentTitles} />
+        </CardBody>
```

Comment:
It would be best to add the conditional logic to render `<Attachments>` around `<CardBody>`, so as to not render an empty element
```suggestion
        {attachmentTitles && (
          <CardBody>
            <Attachments attachments={attachmentTitles} />
          </CardBody>
        )}
```